### PR TITLE
Release v4.2.2

### PR DESCRIPTION
**Pull request by dvaldivia**
- MinIO: `minio/minio:RELEASE.2021-08-20T18-32-01Z`
- Console: `v0.9.4`
- KES: `v0.16.1`

Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

**Review by harshavardhana — APPROVED** [review]

**Review by Alevsk — APPROVED** [review]

**Review by bexsoft — DISMISSED** [review]

**Review by harshavardhana — CHANGES_REQUESTED** [review]
Wait we are making a new MinIO release

**Comment by Alevsk** [comment]
> Wait we are making a new MinIO release

ill test with the new MinIO release then

**Comment by dvaldivia** [comment]
@harshavardhana I think @Alevsk wants to make another operator release later today xD

**Review by Alevsk — DISMISSED** [review]

**Review by bexsoft — DISMISSED** [review]

File: docs/crd.adoc [hunk 1]
```diff
@@ -4,10 +4,10 @@
 [id="{p}-api-reference"]
 == API Reference
 
-:minio-image: https://hub.docker.com/r/minio/minio/tags[minio/minio:RELEASE.2021-08-17T20-53-08Z]
-:kes-image: https://hub.docker.com/r/minio/kes/tags[minio/kes:v0.15.1]
+:minio-image: https://hub.docker.com/r/minio/minio/tags[minio/minio:RELEASE.2021-08-20T18-32-01Z]
```
Comment (harshavardhana) [inline docs/crd.adoc:7]:
```suggestion
:minio-image: https://hub.docker.com/r/minio/minio/tags[minio/minio:RELEASE.2021-08-25T00-41-18Z]
```

File: docs/templates/asciidoctor/gv_list.tpl [hunk 2]
```diff
@@ -7,10 +7,10 @@
 [id="{p}-api-reference"]
 == API Reference
 
-:minio-image: https://hub.docker.com/r/minio/minio/tags[minio/minio:RELEASE.2021-08-17T20-53-08Z]
-:kes-image: https://hub.docker.com/r/minio/kes/tags[minio/kes:v0.15.1]
+:minio-image: https://hub.docker.com/r/minio/minio/tags[minio/minio:RELEASE.2021-08-20T18-32-01Z]
```
Comment (harshavardhana) [inline docs/templates/asciidoctor/gv_list.tpl:10]:
```suggestion
:minio-image: https://hub.docker.com/r/minio/minio/tags[minio/minio:RELEASE.2021-08-25T00-41-18Z]
```

File: examples/tenant-encryption.yaml [hunk 3]
```diff
@@ -108,7 +108,7 @@ metadata:
 
 spec:
   ## Registry location and Tag to download MinIO Server image
-  image: minio/minio:RELEASE.2021-08-17T20-53-08Z
+  image: minio/minio:RELEASE.2021-08-20T18-32-01Z
```
Comment (harshavardhana) [inline examples/tenant-encryption.yaml:111]:
```suggestion
  image: minio/minio:RELEASE.2021-08-25T00-41-18Z
```

File: examples/tenant-lite.yaml [hunk 4]
```diff
@@ -47,7 +47,7 @@ metadata:
 
 spec:
   ## Registry location and Tag to download MinIO Server image
-  image: minio/minio:RELEASE.2021-08-17T20-53-08Z
+  image: minio/minio:RELEASE.2021-08-20T18-32-01Z
```
Comment (harshavardhana) [inline examples/tenant-lite.yaml:50]:
```suggestion
  image: minio/minio:RELEASE.2021-08-25T00-41-18Z
```

File: examples/tenant-log-search.yaml [hunk 5]
```diff
@@ -31,7 +31,7 @@ metadata:
 
 spec:
   ## Registry location and Tag to download MinIO Server image
-  image: minio/minio:RELEASE.2021-08-17T20-53-08Z
+  image: minio/minio:RELEASE.2021-08-20T18-32-01Z
```
Comment (harshavardhana) [inline examples/tenant-log-search.yaml:34]:
```suggestion
  image: minio/minio:RELEASE.2021-08-25T00-41-18Z
```

File: examples/tenant-pod-security-policy.yaml [hunk 6]
```diff
@@ -76,7 +76,7 @@ metadata:
 #  name: my-custom-scheduler
 spec:
   ## Registry location and Tag to download MinIO Server image
-  image: minio/minio:RELEASE.2021-08-17T20-53-08Z
+  image: minio/minio:RELEASE.2021-08-20T18-32-01Z
```
Comment (harshavardhana) [inline examples/tenant-pod-security-policy.yaml:79]:
```suggestion
  image: minio/minio:RELEASE.2021-08-25T00-41-18Z
```

File: examples/tenant-prometheus.yaml [hunk 7]
```diff
@@ -31,7 +31,7 @@ metadata:
 
 spec:
   ## Registry location and Tag to download MinIO Server image
-  image: minio/minio:RELEASE.2021-08-17T20-53-08Z
+  image: minio/minio:RELEASE.2021-08-20T18-32-01Z
```
Comment (harshavardhana) [inline examples/tenant-prometheus.yaml:34]:
```suggestion
  image: minio/minio:RELEASE.2021-08-25T00-41-18Z
```

File: examples/tenant-tiny.yaml [hunk 8]
```diff
@@ -47,7 +47,7 @@ metadata:
 
 spec:
   ## Registry location and Tag to download MinIO Server image
-  image: minio/minio:RELEASE.2021-08-17T20-53-08Z
+  image: minio/minio:RELEASE.2021-08-20T18-32-01Z
```
Comment (harshavardhana) [inline examples/tenant-tiny.yaml:50]:
```suggestion
  image: minio/minio:RELEASE.2021-08-25T00-41-18Z
```

File: examples/tenant-with-autocert-and-custom-certs-encryption-enabled.yaml [hunk 9]
```diff
@@ -101,7 +101,7 @@ metadata:
 
 spec:
   ## Registry location and Tag to download MinIO Server image
-  image: minio/minio:RELEASE.2021-08-17T20-53-08Z
+  image: minio/minio:RELEASE.2021-08-20T18-32-01Z
```
Comment (harshavardhana) [inline examples/tenant-with-autocert-and-custom-certs-encryption-enabled.yaml:104]:
```suggestion
  image: minio/minio:RELEASE.2021-08-25T00-41-18Z
```

File: examples/tenant-with-autocert-and-ldap.yaml [hunk 10]
```diff
@@ -147,7 +147,7 @@ metadata:
 
 spec:
   ## Registry location and Tag to download MinIO Server image
-  image: minio/minio:RELEASE.2021-08-17T20-53-08Z
+  image: minio/minio:RELEASE.2021-08-20T18-32-01Z
```
Comment (harshavardhana) [inline examples/tenant-with-autocert-and-ldap.yaml:150]:
```suggestion
  image: minio/minio:RELEASE.2021-08-25T00-41-18Z
```

File: examples/tenant-with-autocert-encryption-disabled.yaml [hunk 11]
```diff
@@ -47,7 +47,7 @@ metadata:
 
 spec:
   ## Registry location and Tag to download MinIO Server image
-  image: minio/minio:RELEASE.2021-08-17T20-53-08Z
+  image: minio/minio:RELEASE.2021-08-20T18-32-01Z
```
Comment (harshavardhana) [inline examples/tenant-with-autocert-encryption-disabled.yaml:50]:
```suggestion
  image: minio/minio:RELEASE.2021-08-25T00-41-18Z
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**